CDCgov/prime-reportstream
# Flexion/jgilmore/16196-add-environment-variable-for-default encoding

This PR Adds Environment variable "JAVA_OPTS" with value ""-Dfile.encoding=UTF-8" to all environments so the deployed environments are consistent with local environments which have the JAVA_OPTS set in prime-router/local.settings.json

Test Steps:
1. Deploy branch to demo environment (currently deployed to demo1)
2. Verify that "JAVA_OPTS" env variable is set with value "-Dfile.encoding=UTF-8"

## Changes
- Add Environment variable "JAVA_OPTS" with value ""-Dfile.encoding=UTF-8" to all env terraform files
- Add Integration test for HL7 to FHIR for ORU

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x ] Added tests?

### Process
- [x ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #16196
